pyproject.toml

```diff
@@ -22,4 +22,6 @@ requires-python = ">=3.11"
 dependencies = [
     "numba>=0.61.2",
     "numpy",
+    "torch",
```

Comment:
Suggest dropping this. PyTorch is added to the installation instructions in #18.
```suggestion
```

Reply:
If you could review that one, that'd be great.